decidim-comments/spec/queries/sorted_comments_spec.rb

```diff
@@ -69,17 +69,6 @@ module Decidim::Comments
       end
     end
 
-    context "when the comment is hidden" do
-      before do
-        moderation = create(:moderation, reportable: comment, participatory_space: comment.component.participatory_space, report_count: 1, hidden_at: Time.current)
-        create(:report, moderation: moderation)
-      end
-
-      it "is not included in the query" do
```

Comment:
Why removing this spec? Are hidden comments available through the API?

Reply:
The moderated comments are not displayed in the API . 
The `SortedComments` is used in multiple places like the `CommentsCells` or `CommentsController`

Reply:
In order to display the moderated comments we need to remove .not_hidden scope from SortedComments, and use it where we need it. This test is checking that Sorted Comments is not returning any moderated comments, which is not the case anymore.

Reply:
Giving it a second thought, I'd prefer to leave the spec, but of course changing it to 
```ruby
      it "is included in the query" do
       expect(subject.query).not_to be_empty
```
To catch possible regressions and to communicate/document better the change in the SortedComments behavior. Can you change that please?